client/state/posts/actions/normalize-post.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Internal dependencies
+ */
+import { decodeEntities } from 'lib/formatting';
+
+function getParentId( post ) {
+	if ( ! post || ! post.parent ) {
+		return null;
+	}
+
+	if ( post.parent.ID ) {
+		return post.parent.ID;
+	}
+
+	return post.parent;
+}
+
+function getPageTemplate( post ) {
+	if ( ! post || ! post.page_template || post.page_template === 'default' ) {
+		return '';
+	}
+	return post.page_template;
+}
+
+export function normalizePost( post ) {
```

Comment:
This doesn't seem to be an action. Should we move it to some other file, utils.js or something?

Reply:
Right, it's not; it's a shared util used by several actions. Renamed and moved to `utils/`.